pkg/cloudprovider/providers/gce/kms.go

```diff
@@ -129,19 +134,23 @@ func (gce *GCECloud) getGCPCloudKMSService(config io.Reader) (envelope.Service,
 	return service, nil
 }
 
-// Decrypt decrypts a base64 representation of encrypted bytes.
+// Decrypt decrypts a base64 representation of (key-version-used-to-encrypt + encrypted bytes).
```

Comment:
("<key-version-used-to-encrypt>:<encrypted-bytes>")

Reply:
Github hid some of my comment. It was supposed to be:

`("<key-version-used-to-encrypt>:<encrypted bytes>")`